src/dyz.c

```diff
@@ -1,6 +1,8 @@
 #include "lua.h"
 #include "lualib.h"
 #include "lauxlib.h"
+#include "config.h"
```

Comment:
This would be unneeded by linking the Lua sources as `.o` files containing bytecode, and Autotools only would need to generate a Make snippet.